### DISCUSSION TITLE
Does sveltekit support http2?

### DISCUSSION THREAD

**Discussion by AlbertMarashi** [sveltejs/kit#10055] [Q&A] [answered]
### Describe the problem

All of my requests on my app deployed on Vercel are using http 1.1

Is this a vercel issue, or sveltekit issue?

### Describe the proposed solution

Need to use HTTP2 for increased speed & performance

### Importance

would make my life easier

**Comment by gtm-nayan** [comment, marked as answer]
If you check any of the preview deployments on the PRs in this repo that are hosted on Vercel you'll see that they're using HTTP 2.

**Reply by AlbertMarashi** [reply]
Sorry I just realised that firefox was hiding the fact that it was using http2. I had to click on a request in the inspector to see `http/1.1+h2`
